outlines/generate/__init__.py

```diff
@@ -33,6 +31,12 @@ class APIGenerator:
     model: APIModel
     output_type: Optional[Union[Json, List, Choice, Regex]] = None
 
+    def __post_init__(self):
```

Comment:
Wouldn't `__init__` work there?

Reply:
I don't want to interfere with the `__init__` generated by `dataclass`